package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "template",
```

Completar la información
```suggestion
  "name": "Burguer queen",
```